apps/test/unit/templates/projects/ProjectHeaderTest.js

```diff
@@ -10,11 +10,13 @@ describe('ProjectHeader', () => {
       <ProjectHeader canViewAdvancedTools={true} projectCount={10000000} />
     );
 
-    // Note because we don't have a locale in the test we get an unformatted number
-    // Example 10000000 instead of 10,000,000
+    // Expect the number to be formatted to the appropriate client locale
+    // (which can change depending on the browser running this test)
+    const localeFormattedCount = Number(10000000).toLocaleString();
```

Comment:
PhantomJS doesn't have a default locale set, and will render the number unchanged when you call `toLocaleString()`.  ChromeHeadless actually detects locale and will render the number appropriately.
This way, the test checks that the locale transform happens according to the browser's configuration, and passes on both browsers.